src/workflows-app/SubmissionDetails.test.js

```diff
@@ -33,9 +35,30 @@ jest.mock('src/libs/feature-previews', () => ({
 
 jest.mock('src/libs/config', () => ({
   ...jest.requireActual('src/libs/config'),
-  getConfig: jest.fn().mockReturnValue({ cbasUrlRoot, cromwellUrlRoot }),
+  getConfig: jest.fn().mockReturnValue({ cbasUrlRoot, cromwellUrlRoot, wdsUrlRoot }),
 }));
 
+jest.mock('react-virtualized', () => {
```

Comment:
Could you leave a comment here explaining why this is necessary? Want to make it slightly easier for the next person in your shoes.